docs/content/.vuepress/styles/palette.scss

```diff
@@ -8,7 +8,7 @@ $font-family: 'DM Sans';
 $font-family-heading: 'Sharp Sans, sans-serif';
 
 $bg-color: (
-  light: $zinc-100,
+  light: #fcfdfc,
```

Comment:
Is there a `$zinc-50`? Light themes tend to need these fractional shades, but sometimes they’re not available…

Reply:
There sure is. `zinc-50` on left, current site on right:
![image](https://github.com/dxos/dxos/assets/1310143/bc7c729d-a437-4c94-8698-bf1b0ab9f605)

Reply:
Are all the other surfaces also using zinc-50? They look indiscernable, and now the message box in the main area looks out-of-flow.

I think elevation effects (i.e. shadows) are way overapplied in this theme, def think we should flatten things a bit.

Reply:
> Are all the other surfaces also using zinc-50?

If you mean the sidebar and nav bar then yes, they use the background too. The message box is plain `#ffffff`, but the way it now blends into the background is a good example of how changing the background will have lots of other effects, so we should do it once and get it right.

> I think elevation effects (i.e. shadows) are way overapplied in this theme, def think we should flatten things a bit.

Definitely makes sense to me.